attestation-service/attestation-service/src/policy_engine/opa/mod.rs

```diff
@@ -29,31 +30,16 @@ pub struct GoString {
 
 #[derive(Debug)]
 pub struct OPA {
-    policy_dir_path: PathBuf,
+    policy_map: PolicyMap,
 }
 
 impl OPA {
-    pub fn new(work_dir: PathBuf) -> Result<Self> {
-        let mut policy_dir_path = work_dir;
-
-        policy_dir_path.push("opa");
-        if !policy_dir_path.as_path().exists() {
-            fs::create_dir_all(&policy_dir_path)
-                .map_err(|e| anyhow!("Create policy dir failed: {:?}", e))?;
-        }
-
-        let mut default_policy_path = PathBuf::from(
-            &policy_dir_path
-                .to_str()
-                .ok_or_else(|| anyhow!("Policy DirPath to string failed"))?,
-        );
-        default_policy_path.push("default.rego");
-        if !default_policy_path.as_path().exists() {
-            let policy = std::include_str!("default_policy.rego").to_string();
-            fs::write(&default_policy_path, policy)?;
-        }
+    pub fn new() -> Result<Self> {
+        let default_policy = std::include_str!("default_policy.rego");
+        let mut policy_map = HashMap::new();
```

Comment:
The downside to this is that the policies will disappear once the AS is stopped. Is that going to be an issue? In theory we could have multiple storage backends for the policies, but that might be overkill.